app.py

```diff
@@ -1,247 +1,64 @@
-import os, sys, base64, json, functools
-from furl import furl
-import boto3
-from botocore.vendored import requests
-from chalice import Chalice, Response
-import jwt, yaml
+import os
+import sys
 
-from cryptography.hazmat.primitives.asymmetric import rsa
-from cryptography.hazmat.backends import default_backend
+import yaml
+from botocore.vendored import requests
+from chalice import Response as chalice_response
+from flask import json
 
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'chalicelib'))
 sys.path.insert(0, pkg_root)  # noqa
 
-from fusillade import Config, User, Role, Group, directory
+from fusillade import Config
+from fusillade.api import FusilladeServer
 
-iam = boto3.client("iam")
-app = Chalice(app_name='fusillade')
-app.debug = True
 
 with open(os.path.join(pkg_root, "service_config.json")) as fh:
     service_config = yaml.load(fh.read())
 
+swagger_spec_path = os.path.join(pkg_root, "fusillade-api.yml")
+app = FusilladeServer(app_name='fusillade', swagger_spec_path=swagger_spec_path)
+Config.app = app
+
 
 @app.route("/")
 def serve_swagger_ui():
     with open(os.path.join(pkg_root, "index.html")) as fh:
         swagger_ui_html = fh.read()
-    return Response(status_code=200,
-                    headers={"Content-Type": "text/html"},
-                    body=swagger_ui_html)
+    return chalice_response(status_code=requests.codes.ok,
+                            headers={"Content-Type": "text/html"},
+                            body=swagger_ui_html)
 
 
-@app.route('/swagger.json')
-def serve_swagger_definition():
-    with open(os.path.join(pkg_root, "swagger.yml")) as fh:
-        swagger_defn = yaml.load(fh.read())
-    return swagger_defn
+@app.route("/version")
+def version():
+    data = {
+        'version_info': {
+            'version': 0.0
```

Comment:
consider injecting service version in https://github.com/HumanCellAtlas/fusillade/blob/master/build_chalice_config.sh and using os.environ["SERVICE_VERSION"] here or similar

Reply:
Create tickets to update this endpoint https://github.com/HumanCellAtlas/fusillade/issues/97